lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -40,7 +40,7 @@ class UsesOptimizedImages extends Audit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Avoids unoptimized images',
+      description: 'Unoptimized Images',
```

Comment:
Images -> image

sentence case everywhere.  PSI does that too.

<img width="528" alt="screen shot 2017-02-14 at 7 01 26 pm" src="https://cloud.githubusercontent.com/assets/238208/22959068/0b962138-f2e8-11e6-9f1d-69574a8c4546.png">